### ISSUE TITLE
Game Engine useful links and convo

### ISSUE THREAD

**Issue by taylorajez** [GamesTwoSpring2015/Project2.5D#14]
@sofoj1 @daihoshi @TJpenguin15

**Comment by taylorajez** [comment]
Hey guys so as of now Processing is the backbone of our project. https://processing.org/ What do you guys think?

**Comment by taylorajez** [comment]
P.5 is also an option. http://p5js.org/

**Comment by sofoj1** [comment]
I think Processing sounds like the best bet right now.

**Comment by taylorajez** [comment]
Agreed, it doesn't look like we need anything beefier for what we are doing.
